app/models/user.rb

```diff
@@ -7,12 +7,19 @@ class User < ApplicationRecord
   has_many :comments, foreign_key: :author_id
   has_many :likes, foreign_key: :author_id
 
+  validates :name, presence: true
+  validates :posts_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+
   # before_validation :set_default_posts_counter
 
   def recent_posts
     posts.order(created_at: :desc).limit(3)
   end
 
+  def admin?
+    role == 'admin'
+  end
+
```

Comment:
Kudos for creating the `user.admin?` method inside the user model. Fantastic job!!! 👏🏾💯